src/pages/AppList.svelte

```diff
@@ -1,16 +1,25 @@
 <script>
+  import FilterBox from "../components/FilterBox.svelte";
```

Comment:
Let's actually call this a FilterInput, as FilterBox is a bit ambiguous (you'll also need to rename the component).

```suggestion
  import FilterInput from "../components/FilterInput.svelte";
```